packages/@storefront/breadcrumbs/src/breadcrumbs/index.ts

```diff
@@ -58,13 +68,27 @@ class Breadcrumbs {
 
   updateCorrectedQuery = (correctedQuery: string) => this.set({ correctedQuery });
 
+  /**
+   * @deprecated
+   */
   updateFields = () => this.set({ fields: this.getFields(this.state.navigationsSelector()) });
 
+  // tslint:disable-next-line max-line-length
+  updateSelectedNavigations = () => this.set({ selectedNavigations: this.getSelectedNavigations(this.state.navigationsSelector()) });
```

Comment:
If we consider `// tslint:disable-next-line max-line-length` a no-no, then:
```suggestion
  updateSelectedNavigations = () => this.set({
    selectedNavigations: this.getSelectedNavigations(this.state.navigationsSelector())
  });
```

Reply:
I would just do 
```suggestion
  updateSelectedNavigations = () => 
    this.set({ selectedNavigations: this.getSelectedNavigations(this.state.navigationsSelector()) });
```

Reply:
Will do, my bad I missed this one with the latest feedback.